<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharePage</class>
 <widget class="QWidget" name="SharePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="giveButton">
       <property name="text">
        <string>Give</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="visitButton">
       <property name="text">
        <string>Visit Website</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showQRCode">
       <property name="text">
        <string>Show &amp;QR Code</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/qrcode</normaloff>:/icons/qrcode</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>RIght-click for options</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="copyToClipboard">
       <property name="toolTip">
        <string>Copy the currently selected address to the system clipboard</string>
       </property>
       <property name="text">
        <string>&amp;Copy Address</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="verifyMessage">
       <property name="toolTip">
        <string>Verify a message to ensure it was signed with a specified 2GiveCoin address</string>
       </property>
       <property name="text">
        <string>&amp;Verify Message</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/transaction_0</normaloff>:/icons/transaction_0</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="toolTip">
        <string>Delete the currently selected address from the list</string>
       </property>
       <property name="text">
        <string>Refresh Campaigns</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
